libs/ledger-live-common/src/exchange/swap/hooks/useProviderRates.ts

```diff
@@ -1,5 +1,5 @@
 import { useCallback, useEffect, useReducer, useState } from "react";
-import { getExchangeRates } from "..";
+import { dex, getExchangeRates } from "..";
```

Comment:
⚠️ I would not update live-common for this task (or at least not for now) since it will cause issues with LLM that does not handle dex providers in the swap as of today.

Also, there is some logic duplication since you handle the list of dex providers in LLC and in LLD
There is therefore no single source of truth.

I would rather only handle this static dex list on the UI side (LLD as of today), like it was done for the original banner.
The main difference being that now you are appending these 2 states providers entries (they are actually not rates per say) to the dynamic list of providers returned by the API.

Which, by the way, seems to already be what you are doing in your `Rates.index.tsx` component (cf. [here](https://github.com/LedgerHQ/ledger-live/blob/673a66eec8c6996fa5f71188683a3e38be5e1838/apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/Rates/index.jsx#L155))

Reply:
Any logic has been removed from LLC